.readthedocs.yaml

```diff
@@ -1,15 +1,28 @@
 version: 2
 
-python:
-   # Revise build version once more dependencies support python 3.9+
-   version: 3.8
-   system_packages: false
-   install:
-     - method: pip
-       path: .
-       extra_requirements:
-          - docs
+build:
+  os: "ubuntu-20.04"
+  tools:
+    python: "mambaforge-4.10"
+
+conda:
+  environment: docs/rtd_environment.yaml
```

Comment:
Would be nice to not rely on any conda stuff for CI. Do you plan to have this temporarily, or this is now our solution for docs builds?

Reply:
Great question.  I did this because it seemed the most direct way to get RTD to build again.  I'm totally happy to return to a non-conda approach.  I was unable to figure one out - graphviz specifically caused problems.

Reply:
And did you find out why it stopped working the first place? As grapviz wasn't really causing issues recently

Reply:
This error started appearing spontaneously:
https://readthedocs.org/projects/astroquery/builds/20444648/
"Could not import extension sphinx.builders.linkcheck (exception: urllib3 v2.0 only supports OpenSSL 1.1.1+, currently the 'ssl' module is compiled with OpenSSL 1.0.2n  7 Dec 2017. See: https://github.com/urllib3/urllib3/issues/2168)"

My best read was that there is a problem caused by using old python3.8.  We then had to upgrade to the new RTD configuration system, and then _that_ caused problems with graphviz until I switched to the astropy conda-based version.

Reply:
yeap, but graphviz can be installed into the image directly without adding conda into the picture. #2732 should fix it.